### DISCUSSION TITLE
Connect to PostgreSQL as supabase_admin

### DISCUSSION THREAD

**Discussion by iftachhh** [supabase/supabase#8877] [Questions]
Hey there, I'm trying connect to my newly created database as supabase_admin, but with the password generated on onboarding I can only connect as "postgres" user, is there any way for me to run queries as admin from external database gui?

**Comment by seb-lewis** [comment]
Also very curious about this, especially as it's not possible to change pgaudit log settings without higher privilege!

**Comment by xEverth** [comment]
I also encountered privilege issues when trying to set pgaudit log_parameter "on" to log the parameters to the RPCs. Does anyone know if this specific setting can be adjusted as the postgres user, via the dashboard, via the API or in any other way?

**Comment by nelminanoc98** [comment]
is this possible? anyone able to do this?

**Reply by GaryAustin1** [reply]
Not on hosted systems
